localization/english/wc_game_concepts_l_english.yml

```diff
@@ -0,0 +1,24 @@
+﻿ l_english:
+     game_concept_skirmishers:0 "Skirmishers"
+     game_concept_skirmisher:0 "Skirmisher"
+     game_concept_skirmishers_desc:0 "Skirmishers are [men_at_arms|E] who are lightly armed and skilled in harassing the enemy.\n\nWhile each kind of Skirmisher is unique, they generally [counter|E] [heavy_infantry|E] and [caster|E], and are in turn Countered by [archers|E] and [massive_infantry|E]."
+     game_concept_archers:0 "Archers"
+     game_concept_archer:0 "Archers"
+     game_concept_archers_desc:0 "Archers are [men_at_arms|E] who fight on foot with ranged weapons.\n\nWhile each kind of Archer is unique, they generally [counter|E] [skirmishers|E] and [light_aerial|E], and are in turn Countered by [heavy_cavalry|E] and [light_cavalry|E]."
+     game_concept_light_cavalry:0 "Light Cavalry"
+     game_concept_light_cavalry_desc:0 "Light Cavalry are mounted [men_at_arms|E] who excel at scouting and pursuing the enemy.\n\nWhile each kind of Light Cavalry is unique, they generally [counter|E] [archers|E] and [caster|E], and are in turn Countered by [spearmen|E].\n\nLight Cavalry are also often strong on flat [terrain|E].\n\nIn comparison with [heavy_cavalry|E], they are usually better at [pursuit|E] and [screen|E]."
+     game_concept_heavy_cavalry:0 "Heavy Cavalry"
+     game_concept_heavy_cavalry_desc:0 "Heavy Cavalry are mounted, heavily armored [men_at_arms|E]\n\nWhile each kind of Heavy Cavalry is unique, they generally [counter|E] [archers|E] and [caster|E], and are in turn Countered by [spearmen|E] and [massive_aerial|E].\n\nHeavy Cavalry are also often strong on flat [terrain|E].\n\nIn comparison with [light_cavalry|E], they are usually stronger at raw [damage|E] and [toughness|E]."
+     game_concept_spearmen:0 "Spearmen"
+     game_concept_spearmen_desc:0 "Spearmen are [men_at_arms|E] who fight on foot with various polearms.\n\nWhile each kind of Spearman is unique, they generally [counter|E] [heavy_cavalry|E], [light_cavalry|E] and [massive_infantry|E], and are in turn Countered by [heavy_infantry|E] and [caster|E].\n\nSpearmen are also often strong on hilly and mountainous [terrain|E]."
+     game_concept_heavy_infantry:0 "Heavy Infantry"
+     game_concept_heavy_infantry_desc:0 "Heavy Infantry are [men_at_arms|E] who fight on foot, with heavier armor and weapons.\n\nWhile each kind of Heavy Infantry is unique, they generally [counter|E] [spearmen|E], and are in turn Countered by [skirmishers|E] and [caster|E]."
```

Comment:
These are vanilla replacements. You must put them somewhere in `Warcraft-Guardians-of-Azeroth-2\localization\replace\english` if you want them to work correctly.